frontend/src/e2e-playwright/specs/4_finance/invoices.spec.ts

```diff
@@ -168,6 +172,7 @@ describe('Invoices', () => {
     await invoicesPage.freeTextFilter(adultFixtureWihtoutSSN.firstName)
     await invoicesPage.assertInvoiceCount(1)
     await invoicesPage.toggleAllInvoices(true)
+    await invoicesPage.assertInvoiceCount(1)
```

Comment:
I don't understand the idea behind adding this check. The data isn't realoaded after `await invoicesPage.toggleAllInvoices(true)` so the previous `await invoicesPage.assertInvoiceCount(1)` should be enough.

Reply:
🤦 It's so nice to have PR reviews.